Extend Cypress documentation (#3054)
corona-warn-app/cwa-website: README.md

Best practice is to use `data-e2e="your_test_id"` element attributes to select specific elements, eg `cy.get('[data-e2e="cta-button-1"]').click()` instead of `cy.get('.container .infobox a.button').contains('DOWNLOAD').click()`.

|                       | Included in Cypress Test Production?     |  What is it used for?                                                                |
|-----------------------|-|-------------------------------------------------------------------------------------------------------------------------------|

> | check_links.js        |✘ (takes a long time to finish running)   | Detect broken links throughout the website. Result is saved in `logs` folder         |

Reviewer: You don't have to write this, but the reason that `check_links.js` is not included in the production test is because it does not reliably pass. It depends on external website over which we have no control.

> | app_to_web.js         |✓                                         | Detect broken links throughout the website coming from cwa-webserver                |

Reviewer: "Detect broken links throughout the website coming from cwa-webserver"

Change to:

"Check that all links to the website used by the Android and iOS apps are working and that content is correctly displayed."

Author: Changed!

> | app_to_web.js         |✓                                         | Detect broken links throughout the website comming from cwa-webserver                |

Reviewer 2: ```suggestion
| app_to_web.js         |✓                                         | Detect broken links throughout the website coming from cwa-webserver                |
```
Spelling mistake: comming -> coming

| applink_spec.js       |✓                                         | Detect broken links or missing OS badges to download mobile app in home page         |

> | blog_spec.js          |✓                                         | Check if blog archive is accesible                                                   |

Reviewer 2: ```suggestion
| blog_spec.js          |✓                                         | Check if blog archive is accessible                                                   |
```
Spelling mistake: accesible -> accessible

| check_anchor_links.js |✓                                         | Detect broken anchor links throughout the website. Result is saved in `logs` folder  |
| check_videos.js       |✓                                         | Detect broken videos throughout the website                                          |
| eventRegistration.js  |✓                                         | Checks that all fields in event-registration exist and fill them to create a QR code |
| faq.js                |✓                                         | Simulate a FAQ search and the result for mobile and desktop view                     |
| hotline_spec.js       |✓                                         | Verify that Hotline APP and TAN data is correcly displayed at home page              |

> | mime.js               |✓                                         | Check that CSS and JS files have the correct myme                                    |

Reviewer 2: ```suggestion
| mime.js               |✓                                         | Check that CSS and JS files have the correct mime                                    |
```
Spelling mistake: myme -> mime

> | mime.js               |✓                                         | Check that CSS and JS files have the correct mime                                    |

Reviewer: "Check that CSS and JS files have the correct mime" =>

"Check that CSS and JS files have the correct MIME type"
See https://developer.mozilla.org/en-US/docs/Glossary/MIME_type

Author: Done!

Reviewer: @brifemu 
I didn't necessarily mean to add the link to the README. That was just a reference to confirm that correct spelling. It's fine to leave it in though.

To run all test included in Cypress Test Production execute:
`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headed` => test results are printed in console, also you can see browser's movements

> `npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in console, you can`t see browser's movements

Reviewer 2: ```suggestion
`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in console, you can't see browser's movements
```
I'd suggest to replace ` with ' here for consistency reasons. In markdown the text afterwards is also highlighted in blue because it is considered as the start of a code section, like seen before in the same line

> To run all test included in Cypress Test Production execute:
> `npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headed` => test results are printed in console, also you can see browser's movements
> `npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in console, you can't see browser's movements

Reviewer: ```suggestion
To run all tests included in Cypress Test Production execute:

`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headed` => test results are printed in the console, also you can see browser's movements

`npx cypress run -s 'cypress/integration/*.js' -c baseUrl=https://coronawarn.app --headless --browser chrome` => test results are printed in the console, the browser is not displayed

```

Corrected grammar and added blank lines for easier reading.
